x/collection/keeper/supply.go

```diff
@@ -26,15 +26,12 @@ func (k Keeper) CreateContract(ctx sdk.Context, creator sdk.AccAddress, contract
 		Grantee:    creator.String(),
 	}
 	ctx.EventManager().EmitEvent(collection.NewEventGrantPermTokenHead(eventGrant))
-	for _, permission := range collection.Permission_value {
-		p := collection.Permission(permission)
-		if p == collection.PermissionUnspecified {
-			continue
-		}
+	for i := 1; i < len(collection.Permission_value); i++ {
```

Comment:
It's a little bit reluctant to start from number 1 without any explanation.
Which is better, starting from number 0 as before and continuing in case of number 0 or starting from number 1 as now??

Reply:
I think it's quite common protobuf use case, but it may be better to add some explanation.
```suggestion
	// 0 is "unspecified"
	for i := 1; i < len(collection.Permission_value); i++ {
```